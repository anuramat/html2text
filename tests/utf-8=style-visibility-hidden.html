<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html>
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
	<title>Waternet</title>
	<style type="text/css">
		/* boilerplate */
		#outlook a { padding:0; }
		body { font-family:'Maison Neue',Verdana,Arial,sans-serif; width: 100% !important; margin: 0; padding: 0; -webkit-text-size-adjust: 100%; -ms-text-size-adjust: 100%; }
		.ExternalClass { width:100%; }
		.ExternalClass, .ExternalClass p, .ExternalClass span, .ExternalClass font, .ExternalClass td, .ExternalClass div { line-height: 100%; }   
		#backgroundTable { margin:0; padding:0; width:100% !important; line-height: 100% !important; }
		img { border: none; }
		p { margin: 0 0 1em 0; }
		.preheader { display:none !important; visibility:hidden; opacity:0; color:transparent; height:0; width:0; }
		@media only screen and (max-width: 640px)  {
			.deviceWidth { width: 440px !important; padding: 0; }
		}
		@media only screen and (max-width: 479px) {
			.deviceWidth { width: 280px !important; padding: 0; }
		}
	</style>
</head>
<body >
	<span class="preheader" style="display: none !important; visibility: hidden; opacity: 0; color: transparent; height: 0; width: 0;">Nieuwe drinkwaterrekening &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp
;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>
	<table cellpadding="0" cellspacing="0" border="0" id="backgroundTable">
	
		<tr>
		<td class="openhtml" align="center" style="font-family:'Maison Neue',Verdana,Arial,sans-serif;font-size:13px;line-height: 1.5em" >
			<!--[if mso]><p>
			Kunt u deze mail niet goed lezen? | Klik <a href="https://email.waternet.nl/mailview/default.aspx?msg=foooobar" style="color:#000; text-decoration:underline;">hier</a> voor de online versie
			</p>
			<![endif]-->
			<p>
			<div style="font-family:'Maison Neue',Verdana,Arial,sans-serif;font-size:13px;line-height: 1.5em;mso-hide:all">Kunt u deze mail niet goed lezen? |&nbsp;Klik&nbsp;<a href="https://email.waternet.nl/mailview/default.aspx?msg=fooobar" target="_blank" style="color:#000; text-decoration:underline;">hier</a>&nbsp;voor&nbsp;de&nbsp;online&nbsp;versie&nbsp;
			</p>
		</td>
		</tr> 
	</table>
</body>
</html>


