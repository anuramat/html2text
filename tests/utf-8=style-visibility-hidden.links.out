Kunt u deze mail niet goed lezen? | Klik hier[1] voor de online versie 


###### References ######
   1. https://email.waternet.nl/mailview/default.aspx?msg=fooobar
